### PR TITLE
feat: Improve usage examples in TestPane

### LINKED ISSUE #591: feat: Increase the readability of small samples in the Workbench of the Storybook
### Description

Increase the readability of small samples in the Workbench of the Storybook
I'm using the Workbench sample very well in the Storybook.
The unfortunate thing was that when the button was pressed, the action was not seen on the screen, and even when a new control was added, it did not stand out.

### Solution

- style cleanup
- Rename New TreeNode
- Rename New Panel
- Rename New Statusbar
-  Fix code of new Editor

### PR DESCRIPTION
**Pull request by kiwiwong**
## Description

Improve usage examples in TestPane.

Fixes #591 

## Changes

-   Improve the style of TestPane.
-   Improve the logic of some operations.
-   Improve some display names and content.
-   Add more usage examples.
-   Use Notification to show the results of operations that cannot see changes on the page directly.

### CODE REVIEW

**Review by mortalYoung — APPROVED** [review]
+1

**Comment by wewoor** [comment]
I think @SQLGate not only requires the `TestPane` need to improve, and also needs to provide more samples of `Workbench`?

**Comment by wewoor** [comment]
@kiwiwong 这个 PR 我们修改为只针对 TestPane 的更新就好，针对 #591 我们单独 PR

**Comment by kiwiwong** [comment]
> @kiwiwong 这个 PR 我们修改为只针对 TestPane 的更新就好，针对 #591 我们单独 PR

👌

File: stories/extensions/test/testPane.tsx [hunk 1]
```diff
@@ -6,10 +6,13 @@ import { Select, Option } from 'mo/components/select';
 import { IColorTheme } from 'mo/model/colorTheme';
 import { FileTypes, Float, IEditorTab, TreeNodeModel } from 'mo/model';
 import { ILocale } from 'mo/i18n/localization';
-import { localize } from 'mo/i18n/localize';
 import { Scrollable } from 'mo/components';
 import { randomId } from 'mo/common/utils';
 
+function shortRandomId() {
```
Comment (wewoor) [inline stories/extensions/test/testPane.tsx:12]: 这个跟 randomId 啥区别～

Comment (kiwiwong) [reply stories/extensions/test/testPane.tsx:12]: randomId生成的id太长，页面上展示起来感觉不太好看

**Review by wewoor — APPROVED** [review]
+1

**Pull request merged**